---
layout: page
title: Scholarly Work
---

[//]:# Selected Publications
[//]:# ---

### Tumor Clonal Dynamics

{:.paper}
<span>Elements and Evolutionary Determinants of Genomic Divergence Between Paired Primary and Metastatic Tumors</span>{:.papertitle}  
<span>**R. Sun** and **A.N. Nikolakopoulos**</span>{:.authors}<br>
<span>_PLOS Computational Biology_, 17(3), e1008838, Mar. 2021 </span>{:.journal}
☕<span>**DOI:** [10.1371/journal.pcbi.1008838](https://doi.org/10.1371/journal.pcbi.1008838)</span>{:.doi}

{:.paper}
<span>Characterizing the ecological and evolutionary dynamics of cancer</span>{:.papertitle}  
<span>N. Zahir, **R. Sun**, D. Gallahan, R.A. Gatenby, and C. Curtis</span>{:.authors}  
<span>_Nature Genetics_, vol. 52, no. 8, pp. 759--767, Jul. 2020</span>{:.journal}  
☕<span>**DOI:** [10.1038/s41588-020-0668-4](https://dx.doi.org/10.1038/s41588-020-0668-4)</span>{:.doi}  

{:.paper}
<span>Clonal replacement and heterogeneity in breast tumors treated with neoadjuvant HER2-targeted therapy</span>{:.papertitle}  
<span>J.L. Caswell-Jin, K. McNamara, J.G. Reiter, **R. Sun**, Z. Hu, Z. Ma, J. Ding, C.J. Suarez, S. Tilk, A. Raghavendra, V. Forte, S.F. Chin, H. Bardwell, E. Provenzano, C. Caldas, J. Lang, R. West, D. Tripathy, M.F. Press, and C. Curtis</span>{:.authors}  
<span>_Nature Communications_, vol. 10, no. 1, pp. 657, Dec. 2019</span>{:.journal}  
☕<span>**DOI:** [10.1038/s41467-019-08593-4](https://dx.doi.org/10.1038/s41467-019-08593-4)</span>{:.doi}  

{:.paper}
<span>Quantitative evidence for early metastatic seeding in colorectal cancer</span>{:.papertitle}  
<span>Z. Hu, J. Ding, Z. Ma, **R. Sun**, J.A. Seoane, J. Scott Shaffer, C.J. Suarez, A.S. Berghoff, C. Cremolini, A. Falcone, F. Loupakis, P. Birner, M. Preusser, H.J. Lenz, and C. Curtis</span>{:.authors}  
<span>_Nature Genetics_, vol. 51, no. 7, pp. 1113--1122, Jul. 2019</span>{:.journal}  
☕<span>**DOI:** [10.1038/s41588-019-0423-x](https://dx.doi.org/10.1038/s41588-019-0423-x)</span>{:.doi}  

{:.paper}
<span>Big Bang Tumor Growth and Clonal Evolution</span>{:.papertitle}  
<span>**R. Sun**, Z. Hu, and C. Curtis</span>{:.authors}  
<span>_Cold Spring Harbor Perspectives in Medicine_, pp. a028381, Jul. 2017</span>{:.journal}  
☕<span>**DOI:** [10.1101/cshperspect.a028381](https://dx.doi.org/10.1101/cshperspect.a028381)</span>{:.doi}  

{:.paper}
<span>Between-region genetic divergence reflects the mode and tempo of tumor evolution</span>{:.papertitle}  
<span>**R. Sun**, Z. Hu, A. Sottoriva, T.A. Graham, A. Harpak, Z. Ma, J.M. Fischer, D. Shibata, and C. Curtis</span>{:.authors}  
<span>_Nature Genetics_, vol. 49, no. 7, pp. 1015--1024, Jun. 2017</span>{:.journal}  
☕<span>**DOI:** [10.1038/ng.3891](https://dx.doi.org/10.1038/ng.3891)</span>{:.doi}  

{:.paper}
<span>A population genetics perspective on the determinants of intra-tumor heterogeneity</span>{:.papertitle}  
<span>Z. Hu, **R. Sun**, and C. Curtis</span>{:.authors}  
<span>_Biochimica et Biophysica Acta (BBA) - Reviews on Cancer_, vol. 1867, no. 2, pp. 109--126, Apr. 2017</span>{:.journal}  
☕<span>**DOI:** [10.1016/j.bbcan.2017.03.001](https://dx.doi.org/10.1016/j.bbcan.2017.03.001)</span>{:.doi}


### Analytics in Cancer Genomics

{:.paper}
<span>Integrative genomic profiling of large-cell neuroendocrine carcinomas reveals distinct subtypes of high-grade neuroendocrine lung tumors</span>{:.papertitle}  
<span>J. George, V. Walter, M. Peifer, L.B. Alexandrov, D. Seidel, F. Leenders, L. Maas, C. M{ü}ller, I. Dahmen, T.M. Delhomme, M. Ardin, N. Leblay, G. Byrnes, **R. Sun**, A. De Reynies, A. McLeer-Florin, G. Bosco, F. Malchers, R. Menon, J. Altm{ü}ller, C. Becker, et. al., and R.K. Thomas</span>{:.authors}  
<span>_Nature Communications_, vol. 9, no. 1, pp. 1048, Dec. 2018</span>{:.journal}  
☕<span>**DOI:** [10.1038/s41467-018-03099-x](https://dx.doi.org/10.1038/s41467-018-03099-x)</span>{:.doi}  

{:.paper}
<span>Promoter of lncRNA Gene PVT1 Is a Tumor-Suppressor DNA Boundary Element</span>{:.papertitle}  
<span>S.W. Cho, J. Xu, **R. Sun**, M.R. Mumbach, A.C. Carter, Y.G. Chen, K.E. Yost, J. Kim, J. He, S.A. Nevins, S.F. Chin, C. Caldas, S.J. Liu, M.A. Horlbeck, D.A. Lim, J.S. Weissman, C. Curtis, and H.Y. Chang</span>{:.authors}  
<span>_Cell_, May 2018</span>{:.journal}  
☕<span>**DOI:** [10.1016/j.cell.2018.03.068](https://dx.doi.org/10.1016/j.cell.2018.03.068)</span>{:.doi}

<span>Identification of novel fusion genes in lung cancer using breakpoint assembly of transcriptome sequencing data</span>{:.papertitle}  
<span>L. Fernandez-Cuesta, **R. Sun**, R. Menon, J. George, S. Lorenz, L. Meza-Zepeda, M. Peifer, et. al., S.A. Haas, and R.K. Thomas</span>{:.authors}  
<span>_Genome Biology_, vol. 16, no. 1, pp. 7, Jan. 2015</span>{:.journal}  
☕<span>**DOI:** [10.1186/s13059-014-0558-0](https://dx.doi.org/10.1186/s13059-014-0558-0)</span>{:.doi}

{:.paper}
<span>Frequent mutations in chromatin-remodelling genes in pulmonary carcinoids</span>{:.papertitle}  
<span>L. Fernandez-Cuesta, M. Peifer, X. Lu, **R. Sun**, L. Ozretic, D. Seidel, T. Zander, F. Leenders, J. George, C. M{ü}ller, I. Dahmen, B. Pinther, G. Bosco, K. Konrad, J. Altm{ü}ller, P. N{ü}rnberg, V. Achter, U. Lang, P.M. Schneider, M. Bogus, et. al., and R.K. Thomas</span>{:.authors}  
<span>_Nature Communications_, vol. 5, pp. 3518, Jan. 2014</span>{:.journal}  
☕<span>**DOI:** [10.1038/ncomms4518](https://dx.doi.org/10.1038/ncomms4518)</span>{:.doi}

{:.paper}
<span>CD74-NRG1 fusions in lung adenocarcinoma</span>{:.papertitle}  
<span>L. Fernandez-Cuesta, D. Plenker, H. Osada, **R. Sun**, R. Menon, F. Leenders, S. Ortiz-Cuaran, M. Peifer, M. Bos, et. al., and R.K. Thomas</span>{:.authors}  
<span>_Cancer Discovery_, vol. 4, pp. 415--422, Jan. 2014</span>{:.journal}  
☕<span>**DOI:** [10.1158/2159-8290.CD-13-0633](https://dx.doi.org/10.1158/2159-8290.CD-13-0633)</span>{:.doi}

{:.paper}
<span>Integrative genome analyses identify key somatic driver mutations of small-cell lung cancer.</span>{:.papertitle}  
<span>M. Peifer, L. Fern{á}ndez-Cuesta, M. Sos, J. George, D. Seidel, L. Kasper, D. Plenker, F. Leenders, **R. Sun**, T. Zander, R. Menon, M. Koker, I. Dahmen, C. M{ü}ller, V. Di Cerbo, H.U. Schildhaus, J. Altm{ü}ller, I. Baessmann, C. Becker, B. de Wilde, et. al., S. Haas, and R. Thomas</span>{:.authors}  
<span>_Nature genetics_, vol. 44, no. 10, pp. 1104--1110, Oct. 2012</span>{:.journal}  
☕<span>**DOI:** [doi: 10.1038/ng.2396](https://dx.doi.org/doi: 10.1038/ng.2396)</span>{:.doi}  

{:.paper}
<span>Global gene expression analysis reveals reduced abundance of putative microRNA targets in human prostate tumours.</span>{:.papertitle}  
<span>**R. Sun**, X. Fu, Y. Li, Y. Xie, and Y. Mao</span>{:.authors}  
<span>_BMC genomics_, vol. 10, no. 1, pp. 93, Feb. 2009</span>{:.journal}  
☕<span>**DOI:** [doi: 10.1186/1471-2164-10-93](https://dx.doi.org/doi: 10.1186/1471-2164-10-93)</span>{:.doi}  

{:.paper}
<span>Simultaneously detection of genomic and expression alterations in prostate cancer using cDNA microarray.</span>{:.papertitle}  
<span>M. Jiang, M. Li, X. Fu, Y. Huang, H. Qian, **R. Sun**, Y. Mao, Y. Xie, and Y. Li</span>{:.authors}  
<span>_The Prostate_, vol. 68, no. 14, pp. 1496--1509, Oct. 2008</span>{:.journal}  
☕<span>**DOI:** [doi: 10.1002/pros.20756](https://dx.doi.org/doi: 10.1002/pros.20756)</span>{:.doi}  


### Algorithms for genomic sequence analysis

{:.paper}
<span>Breakpointer: using local mapping artifacts to support sequence breakpoint discovery from single-end reads.</span>{:.papertitle}  
<span>**R. Sun**, M. Love, T. Zemojtel, A.K. Emde, H.R. Chung, M. Vingron, and S. Haas</span>{:.authors}  
<span>_Bioinformatics (Oxford, England)_, vol. 28, no. 7, pp. 1024--1025, Apr. 2012</span>{:.journal}  
☕<span>**DOI:** [doi: 10.1093/bioinformatics/bts064](https://dx.doi.org/doi: 10.1093/bioinformatics/bts064)</span>{:.doi}  

{:.paper}
<span>Detecting genomic indel variants with exact breakpoints in single- and paired-end sequencing data using SplazerS</span>{:.papertitle}  
<span>A.K. Emde, M. Schulz, D. Weese, **R. Sun**, M. Vingron, V. Kalscheuer, S. Haas, and K. Reinert</span>{:.authors}  
<span>_Bioinformatics_, vol. 28, no. 5, pp. 619--627, Mar. 2012</span>{:.journal}  
☕<span>**DOI:** [doi: 10.1093/bioinformatics/bts019](https://dx.doi.org/doi: 10.1093/bioinformatics/bts019)</span>{:.doi}  

{:.paper}
<span>Modeling read counts for CNV detection in exome sequencing data.</span>{:.papertitle}  
<span>M. Love, A. My{š}i{č}kov{á}, **R. Sun**, V. Kalscheuer, M. Vingron, and S. Haas</span>{:.authors}  
<span>_Statistical applications in genetics and molecular biology_, vol. 10, no. 1, Nov. 2011</span>{:.journal}  
☕<span>**DOI:** [doi: 10.2202/1544-6115.1732](https://dx.doi.org/doi: 10.2202/1544-6115.1732)</span>{:.doi}  




